File: Topic Based Problems/01 Fundamentals/Beginner/16 Swap Two Numbers/Java/SwapTwoNumbers.java
/**
 * @file SwapTwoNumbers.java
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Swap Two Numbers in Java using arithmetic operator
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

import java.util.Scanner;

public class SwapTwoNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter first number: ");
        int firstNumber = scanner.nextInt();

        System.out.print("Enter second number: ");
        int secondNumber = scanner.nextInt();

        System.out.println("Before swapping: ");
        System.out.println("First number: " + firstNumber);
        System.out.println("Second number: " + secondNumber);

        firstNumber = firstNumber + secondNumber;
        secondNumber = firstNumber - secondNumber;
        firstNumber = firstNumber - secondNumber;

        System.out.println("After swapping: ");
        System.out.println("First number: " + firstNumber);
        System.out.println("Second number: " + secondNumber);

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/Beginner/16 Swap Two Numbers/Python/SwapTwoNumbers.py
"""
SwapTwoNumbers.py
Author: SharifdotG (sharifmdyousuf007@gmail.com)
Swap Two Numbers in Python using arithmetic operator
Version: 1.0
Date: 2023-09-24
Copyright (c) 2023
"""

firstNumber = int(input("Enter first number: "))
secondNumber = int(input("Enter second number: "))

print("Before swapping:")
print("First number:", firstNumber)
print("Second number:", secondNumber)

firstNumber = firstNumber + secondNumber
secondNumber = firstNumber - secondNumber
firstNumber = firstNumber - secondNumber

print("After swapping:")
print("First number:", firstNumber)
print("Second number:", secondNumber)
